packages/swing-store/src/hasher.js

```diff
@@ -23,10 +23,13 @@ function createSHA256(initial = undefined) {
     done = true;
     return hash.digest('hex');
   }
+  function sample() {
```

Comment:
I was in the midst of writing a reply to the effect of "no, we use it quite a lot" and then went looking for examples and discovered that you're right, we don't use it at all.  I think what I was remembering was debug code, where it proved essential for diagnosing hash aggregation problems when I was working on the commit/abort changes.  Since it *was* useful in that case, I'm inclined to leave it in, but I'm open to being persuaded otherwise.  Can you explain the vulnerability potential that you are concerned about?

Reply:
Chip and I talked through this, and decided that it was fine to leave `sample()` in place.